api/voice-react-native-sdk.api.md

```diff
@@ -139,6 +139,14 @@ namespace AuthorizationErrors {
         name: string;
         solutions: string[];
     }
+    class PayloadSizeExceededError extends TwilioError {
+        constructor(message: string);
+        causes: string[];
+        description: string;
+        explanation: string;
+        name: string;
+        solutions: string[];

```

Comment:
also why are `causes` and `solutions` arrays? Would it be easier for the user to just receive 1 long string instead of an array?